src/board.spec.ts

```diff
@@ -56,6 +56,20 @@ describe('convertTrack', () => {
   });
 });
 
+describe('convertPadToVia', () => {
+  it('should correctly parse a PAD and convert it to a Via', () => {
```

Comment:
Good job here! You may need to change the `expect()` bit after you switch to `getNetId()` (see my comment about that below)